### ISSUE TITLE
[Order Details] VoiceOver: Hidden Elements Should not be Available for Navigation 

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#4977]
In an Order Details page, when navigating by element using VoiceOver (swipe-to-right), a hidden "Fees" element is accessible. And then right after swiping to right on that hidden element, the user is navigated back to the **topmost element** instead of the next element.

## Video

https://user-images.githubusercontent.com/198826/132887729-8445712b-e9f2-4854-8856-ad703fa49b46.mp4

## To Reproduce

1. Navigate to an order with no fees.
2. Turn on VoiceOver
3. Swipe-to-right until you reach the hidden Fees element. VoiceOver will say "Fees" but that is actually not visible. 

## Expectations

- [ ]  The "Fees" or any other hidden element should not be accessible. 
- [ ]  Swiping to right should continue on to the next element in the screen, not the topmost element. 

---

Internal ref: p1631199350210900-slack-C6H8C3G23
